.github/workflows/pr-check.yml

```
@@ -5,27 +5,25 @@
+    uses: ./.github/workflows/setup-env.yml
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
   validate:
+    if: github.base_ref == 'master'
+    needs: setup
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)